accounts/models.py

```diff
@@ -57,6 +57,8 @@ class User(AbstractBaseUser, PermissionsMixin):
         },
     )
     name = models.CharField(max_length=200)
```

Comment:
Can we change `name` to `first_name`?  This may cause problems with the migration as it won't be a nullable field, but as we haven't deployed anywhere, you may as well blat the existing migration and create a new one.

Reply:
Remember also that we do have to import the data from last year. But we can probably easily change the data dump.